frontend/src/turbo/aria-stream-action.ts

```diff
@@ -0,0 +1,19 @@
+import { StreamActions, StreamElement } from '@hotwired/turbo';
+import { announce } from '@primer/live-region-element';
+
+export function registerAriaStreamAction() {
+  StreamActions.aria = function dialogStreamAction(this:StreamElement) {
```

Comment:
What is that needed for?

Reply:
I want to define aria action for turbo, I must explicitly register it by assigning a function to StreamActions.aria. Therefore, when a stream with action="aria" arrives, run this custom function I’m registering as StreamActions.aria.